### DISCUSSION TITLE
Snowpack & Rails

### DISCUSSION THREAD

**Discussion by Twistedben** [andrewmcodes/andrewm-codes-website#37] [AMA] [answered]
Andrew,

After listening to one of the Remote Ruby podcasts, I think I recalled you mentioning that you were running Snowpack in Rails. Is this true and do you have any code you can share/process for doing this? 
Thanks so much.

**Comment by andrewmcodes** [comment, marked as answer]
Sorry for the incredibly late response but I have since purged that code since DHH went a different direction. Snowpack is built on ESbuild so he essentially dropped down to avoid the middle guys.

I still think I could easily do it though, if you're still interested and want to pair on it lmk.

**Comment by Twistedben** [comment]
No problem for the late response, thanks for getting back to me.
Yeah we've been messing with ESBuild and import maps, etc. I think Vite might be an attractive option too. I'll let you know if I start to voyage into this.

**Reply by andrewmcodes** [reply]
I’ve considered switching this site to vite since Astro, the ssg made by the snowpack team, is now using vite as well so not sure what that says about the future of snowpack moving forward. 

But it took me a long time to get it working initially that I know it so well now that I don’t want to give up on it.
